scripts/text_validator.js

```diff
@@ -46,6 +46,10 @@ $(function () {
     }
 
     function _validateText() {
+        // IE HACK - xregexp causes problems in IE11, return true if IE
```

Comment:
I think if you take this out the MR is ready for merging.

Reply:
I'd already undone this and rebased it so this last push should have had the required effect.